src/number_theory/cyclotomic/basic.lean

```diff
@@ -324,6 +325,13 @@ lemma splitting_field_X_pow_sub_one : is_splitting_field K L (X ^ (n : ℕ) - 1)
       n.pos _), is_root.def, eval_sub, eval_pow, eval_C, eval_X, sub_eq_zero]
   end }
 
+instance [ne_zero (n : K)] : is_galois K L :=
```

Comment:
I think this is a dangerous instance, right? Because `n` is not determined by `is_galois K L`. Unless it is determined by some `out_param`.
```suggestion
lemma [ne_zero (n : K)] : is_galois K L :=
```

Reply:
it's definitely not an out-param as `ℚ(ζₙ) = ℚ(ζ₂ₙ)` for odd `n`; so you're right

Reply:
Ah sure, let me correct it.